release/pkg/assets_haproxy.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	anywherev1alpha1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+// GetKindnetdAssets returns the eks-a artifacts for kindnetd
+func (r *ReleaseConfig) GetHaproxyAssets() ([]Artifact, error) {
+	gitTag, err := r.readGitTag(kindProjectPath, r.BuildRepoBranchName)
+	if err != nil {
+		return nil, errors.Cause(err)
+	}
+
+	name := "haproxy"
+	repoName := fmt.Sprintf("kubernetes-sigs/kind/%s", name)
+	tagOptions := map[string]string{
+		"gitTag":      gitTag,
+		"projectPath": kindProjectPath,
+	}
+
+	sourceImageUri, sourcedFromBranch, err := r.GetSourceImageURI(name, repoName, tagOptions)
+	if err != nil {
+		return nil, errors.Cause(err)
+	}
+	releaseImageUri, err := r.GetReleaseImageURI(name, repoName, tagOptions)
+	if err != nil {
+		return nil, errors.Cause(err)
+	}
+
+	imageArtifact := &ImageArtifact{
+		AssetName:         name,
+		SourceImageURI:    sourceImageUri,
+		ReleaseImageURI:   releaseImageUri,
+		Arch:              []string{"amd64"},
```

Comment:
Do we only do amd64 still in these places?

Reply:
yeah for now, but i'm changing that in my refactor PR to include both amd64 and arm64. That would make it easier to change, like updating it in one place vs 20 files.

Reply:
cool